### DISCUSSION TITLE
How to get NativeWind to work with Solito

### DISCUSSION THREAD

**Discussion by adherb** [nativewind/nativewind#107] [Q&A] [answered]
I am trying to get NativeWind to work with a Solito monorepo, but I'm having now luck following the installation guide.

is there any examples of the two working together?

**Comment by marklawlor** [comment, marked as answer]
There is an example project here https://github.com/nandorojo/solito/pull/72 that I intent to merge when I release NativeWind

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
